ceph.spec.in

```diff
@@ -1175,12 +1175,27 @@ fi
 
 #################################################################################
 %if %{with ocf}
+
 %files resource-agents
 %defattr(0755,root,root,-)
-%dir /usr/lib/ocf
-%dir /usr/lib/ocf/resource.d
-%dir /usr/lib/ocf/resource.d/ceph
-/usr/lib/ocf/resource.d/%{name}/*
+# N.B. %{_libexecdir} is set by unconditional %global to %{_exec_prefix}/lib
```

Comment:
Good question. It uses the one specified by `./configure --prefix` like so: https://github.com/ceph/ceph/blob/master/src/ocf/Makefile.am#L4-L7

I will replace `%{_libexecdir}` with `%{_prefix}/lib` so it matches.